ISO C99 and later do not support implicit function declarations

Adding `linux-nvme-sys = "0.3.0"` to Cargo.toml leads to 
```
   Compiling linux-nvme-sys v0.3.0
error: failed to run custom build command for `linux-nvme-sys v0.3.0`

Caused by:
  process didn't exit successfully: `/home/alex/p/sw/sedkit/target/debug/build/linux-nvme-sys-48ecbe2c31970123/build-script-build` (exit status: 101)
  --- stderr
  ./nvme-cli/linux/nvme.h:1390:6: error: call to undeclared function 'unlikely'; ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]
  ./nvme-cli/linux/nvme.h:1390:6: error: call to undeclared function 'unlikely'; ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration], err: true
  thread 'main' panicked at 'Unable to generate bindings: ()', /home/alex/.cargo/registry/src/github.com-1ecc6299db9ec823/linux-nvme-sys-0.3.0/build.rs:14:10
  note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

`-std=c89` as custom Clang arg also does not work, because then it does not recognize `inline` as a keyword.

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "linux-nvme-sys"
-version = "0.3.0"
+version = "0.4.0"
 authors = ["MOZGIII <mike-n@narod.ru>"]
 edition = "2018"
 description = "Rust bindgen for nvme on linux"
@@ -8,7 +8,7 @@ license = "MIT"
 repository = "https://github.com/MOZGIII/linux-nvme-sys"
 
 [dependencies]
-nix = "0.15"
+nix = "0.26"
 
 [build-dependencies]
-bindgen = "0.51"
+bindgen = "0.66"

--- a/wrapper.h
+++ b/wrapper.h
@@ -1,6 +1,8 @@
 #include <stdint.h>
 #include <stdbool.h>
 
+#define unlikely(x) x
+
 #include "nvme-cli/linux/nvme.h"
 #include "nvme-cli/linux/nvme_ioctl.h"
 #include "nvme-cli/linux/lightnvm.h"

--- a/wrapper.h
+++ b/wrapper.h
@@ -1,6 +1,10 @@
 #include <stdint.h>
 #include <stdbool.h>
 
+// In the nvme-cli, the `unlikely` macro is defined at `nvme-cli/nvme.h` -
+// but we do not want to include that, so we inline it instead.
+// This might hve to be changed when the `nvme-cli` submodule is updated to
+// a newer version.
 #define unlikely(x) x
 
 #include "nvme-cli/linux/nvme.h"

